src/components/views/messages/MImageBody.js

```diff
@@ -177,8 +177,8 @@ export default class MImageBody extends React.Component {
         // thumbnail resolution will be unnecessarily reduced.
         // custom timeline widths seems preferable.
```

Comment:
This comment block needs updating. Note that the `dharma skin` is the current skin.